hack/newTargetGen/scaffolding/cmd/newtarget-adapter/main.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2021 TriggerMesh Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"github.com/triggermesh/triggermesh/pkg/targets/adapter/{{.Name}}"
+	pkgadapter "knative.dev/eventing/pkg/adapter/v2"
```

Comment:
nit: This alias could be removed. There is no clash with another `adapter` import, so you can simply use it as `adapter` (without any aliasing, "/v2" is ignored by Go due to module versioning conventions).